drivers/pci/controller/vmd.c

```diff
@@ -790,6 +834,27 @@ static void vmd_configure_cfgbar(struct vmd_dev *vmd)
 		       (resource_size(res) >> 20) - 1,
 		.flags = IORESOURCE_BUS | IORESOURCE_PCI_FIXED,
 	};
+
+	if (vmd_has_pch_rootbus(vmd)) {
```

Comment:
please put declarations inside if:
```suggestion
		u16 ioc_range = 0;
		u16 pch_range = 0;

		pci_read_config_word(vmd->dev, VMD_BUSRANGE0, &ioc_range);
		pci_read_config_word(vmd->dev, VMD_BUSRANGE1, &pch_range);
```